tests/_support/Helper/Acceptance.php

```diff
@@ -6,31 +6,19 @@
 
 class Acceptance extends \Codeception\Module
 {
-    public function sendEmail($to, $subject, $body)
+    public function sendEmail($to, $subject, $body, $encoding = '')
     {
         $phpmailer = new \PHPMailer();
         $phpmailer->isSMTP();
         $phpmailer->Host = '127.0.0.1';
         $phpmailer->Port = 1025;
+        if ($encoding) {
```

Comment:
This is a weak check, lets be more strict:

```suggestion
        if (null !== $encoding) {
```